opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py

```diff
@@ -29,19 +30,55 @@
     ObservableUpDownCounter as APIObservableUpDownCounter,
 )
 from opentelemetry._metrics.instrument import UpDownCounter as APIUpDownCounter
+from opentelemetry.sdk._metrics.aggregation import AggregationTemporality
 from opentelemetry.sdk._metrics.measurement import Measurement
+from opentelemetry.sdk._metrics.metric_reader import MetricReader
+from opentelemetry.sdk._metrics.point import Metric
 from opentelemetry.sdk.util.instrumentation import InstrumentationInfo
 
 
+class MeasurementConsumer(ABC):
+    @abstractmethod
+    def consume_measurement(self, measurement: Measurement) -> None:
+        pass
+
+    @abstractmethod
+    def register_asynchronous_instrument(self, instrument: "_Asynchronous"):
+        pass
+
+    @abstractmethod
+    def collect(
+        self, metric_reader: MetricReader, temporality: AggregationTemporality
+    ) -> Iterable[Metric]:
+        pass
+
+
+class SynchronousMeasurementConsumer(MeasurementConsumer):
+    def consume_measurement(self, measurement: Measurement) -> None:
+        pass
+
+    def register_asynchronous_instrument(
+        self, instrument: "_Asynchronous"
```

Comment:
You need an import for this still, why not just import and use that instead of a string?

Reply:
Sorry, import it from where? `_Asynchronous` is in `instrument.py`